packages/patternfly-3/patternfly-react/src/components/Chart/DonutChart.js

```diff
@@ -15,13 +15,14 @@ const setDonutTitle = obj => {
 
   const { props } = obj;
   const { data, title = {} } = props;
+  const { type, percision = 0 } = title;
   const { columns } = data;
   const sum = columns.reduce((acc, x) => acc + x[1], 0);
   const iMax = colIndexOfMaxValue(columns);
 
-  switch (title.type) {
+  switch (type) {
     case 'percent':
-      primary = `${Math.round((100 * columns[iMax][1]) / sum).toString()}%`;
+      primary = `${((100 * columns[iMax][1]) / sum).toFixed(percision).toString()}%`;
```

Comment:
Check out [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat)
```suggestion
      primary = `${Intl.NumberFormat(null, {style: 'percent', maximumFractionDigits: precision}).format(columns[iMax][1]) / sum)}%`;
```

Reply:
Wouldn't this change the output for existing applications? Perhaps we can leave the existing code as is, unless the  `precision` property is added?